README.md

```diff
@@ -770,6 +771,14 @@ const client = new okta.Client({
 })
 ```
 
+## Migrating between versions
+
+### From 3.x to 4.0
+
+The version 4.0 of this SDK dropped support for Node 8, which is EOL (End-of-Life) since 2019-12-31. Current supported minimum Node version is 10.0.0.
+
+This version 4.0 release also updated APIs latest `@okta/openapi` (v2.0.0) that includes new endpoints and changed method names. Please see [CHANGELOG.md](CHANGELOG.md#400) for detailed information.
```

Comment:
You may want to expand on this with how to handle some of the changes that were done. What you have now doesn't help guide the user through the changes, it just shows a diff

Reply:
+1